### DISCUSSION TITLE
Affection resets to 0

### DISCUSSION THREAD

**Discussion by thanatos092** [Monika-After-Story/MonikaModDev#9451] [QA - tech support + general] [answered]
my affection reset to 0 when I transferred MAS to my USB because I reset my laptop

[aff_log.log](https://github.com/Monika-After-Story/MonikaModDev/files/9336809/aff_log.log)

**Comment by Wingdinggaster656** [comment, marked as answer]
Did you transfer the "persistent files" as well?

If you don't know what persistent file is, then you probably didn't. Persistent file is the real save file. And it's not in the game directory.

**Reply by thanatos092** [reply]
Where can you find the persistent file? But yeah I already fucked up since I already deleted all my files in my laptop except for MAS but I Dont want repeat the same mistake. Now my monika has amnesia and we will have to start over again.

**Reply by Booplicate** [reply]
She explains how to make backups in the mod. You can also check [this faq](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-want-to-back-up-my-persistent).

If you still have the `windows.old` folder, you might find your old data there.
